src/sbl_filing_api/entities/models/dao.py

```diff
@@ -104,19 +94,12 @@ def __str__(self):
         return f"ContactInfo ID: {self.id}, First Name: {self.first_name}, Last Name: {self.last_name}, Address Street 1: {self.hq_address_street_1}, Address Street 2: {self.hq_address_street_2}, Address City: {self.hq_address_city}, Address State: {self.hq_address_state}, Address Zip: {self.hq_address_zip}"
 
 
-class SignatureDAO(Base):
-    __tablename__ = "signature"
-    id: Mapped[int] = mapped_column(primary_key=True, autoincrement=True)
-    signer_id: Mapped[str]
-    signer_name: Mapped[str] = mapped_column(
-        nullable=True
-    )  # Some users may not have populated keycloak first/last name
-    signer_email: Mapped[str]
-    signed_date: Mapped[datetime] = mapped_column(server_default=func.now())
-    filing: Mapped[int] = mapped_column(ForeignKey("filing.id"))
-
-    def __str__(self):
-        return f"ID: {self.id}, Filing: {self.filing}, Signer ID: {self.signer_id}, Signer Name: {self.signer_name}, Signing Date: {self.signed_date}"
+class FilingSignatureDAO(Base):
+    __tablename__ = "filing_signature"
```

Comment:
was just referencing the dao object name, although if we do change the table name here, let's make sure the creation script has the same table name; I think it's out of sync now